java_status.go

```diff
@@ -18,6 +19,7 @@ type statusCmd struct {
 
 	RetryInterval time.Duration `usage:"if retry-limit is non-zero, status will be retried at this interval" default:"10s"`
 	RetryLimit    int           `usage:"if non-zero, failed status will be retried this many times before exiting"`
+	TimeOut       time.Duration `usage:"if non-zero, this is the timeout the ping can take as a maximum"`
```

Comment:
I wonder if it should always have a default such as the following. I'd also tweak it to `Timeout` since flagsfiller would have converted to `--time-out` or `TIME_OUT`
 
```suggestion
	Timeout       time.Duration `usage:"the timeout the ping can take as a maximum" default:"60s"`
```